components/ethernet/include/esp_eth.h

```diff
@@ -45,6 +45,11 @@ typedef enum {
     ETH_MODE_FULLDUPLEX,
 } eth_duplex_mode_t;
 
+#define DMA_RX_BUF_NUM CONFIG_DMA_RX_BUF_NUM
+#define DMA_TX_BUF_NUM CONFIG_DMA_TX_BUF_NUM
+#define DMA_RX_BUF_SIZE 1600
+#define DMA_TX_BUF_SIZE 1600
```

Comment:
Since these are now in global namespace, please prefix these defines with `ETH_`.

Reply:
Fixed.
I've also updated Kconfig - I think the config vars `DMA_RX_BUF_NUM` and `DMA_TX_BUF_NUM` also need to be prefixed.